git_bundled_static.go

```diff
@@ -9,8 +9,8 @@ package git
 #cgo CFLAGS: -DLIBGIT2_STATIC
 #include <git2.h>
 
-#if LIBGIT2_VER_MAJOR != 1 || LIBGIT2_VER_MINOR != 1
-# error "Invalid libgit2 version; this git2go supports libgit2 v1.1"
+#if LIBGIT2_VER_MAJOR != 1 || LIBGIT2_VER_MINOR < 0
+# error "Invalid libgit2 version; this git2go supports libgit2 >= v1.0"
```

Comment:
the test results show that this combination is broken, so we need to have this be:

```cpp
#if LIBGIT2_VER_MAJOR != 1 || LIBGIT2_VER_MINOR < 1
# error "Invalid libgit2 version; this git2go supports libgit2 >= v1.1"
```

and since we need to update the CI anyways every time there's a new release to explicitly change the ref, might as well have this be:

```suggestion
#if LIBGIT2_VER_MAJOR != 1 || LIBGIT2_VER_MINOR < 1 || LIBGIT2_VER_MINOR > 1
# error "Invalid libgit2 version; this git2go supports libgit2 between v1.1.0 and v1.1.0"
```

which is... exactly the same as what we have right now. BUT the difference is that in the other branches (say, v30) it'll be:

```cpp
#if LIBGIT2_VER_MAJOR != 1 || LIBGIT2_VER_MINOR < 0 || LIBGIT2_VER_MINOR > 1
# error "Invalid libgit2 version; this git2go supports libgit2 between v1.0.0 and v1.1.0"
```

Reply:
I don't think that an upper bound on the supported minor version is really strictly needed right now, but committed it anyhow as you suggested.

> the test results show that this combination is broken

Yep, only had that in for testing if the build would fail.